apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1263,7 +1263,8 @@ export class PortfolioService {
         [
           new EmergencyFundSetup(
             this.exchangeRateDataService,
-            userSettings.emergencyFund
+            userSettings.emergencyFund ||
```

Comment:
I will change this, because `EmergencyFundSetup()` expects a `number` instead of a `boolean`.